app/src/main/java/com/example/fitjournal/library/presentation/screen/library/model/LibraryWorkoutItem.kt

```diff
@@ -1,6 +1,8 @@
 package com.example.fitjournal.library.presentation.screen.library.model
 
+import com.example.fitjournal.core.presentation.model.enums.WorkoutTypeEnum
+
 data class LibraryWorkoutItem(
```

Comment:
This looks exactly like a data class object I have in my Pr, I wonder if we should make this part of Core and have it in a place more accessible for the whole app.

Reply:
I can move it to core -> Presentation -> model -> Ui model

Reply:
Move it, but give it a more generic name, because that model is not just for library it will be also used for the Journal Entries

Reply:
Actually never mind, you also have a RealmWorkoutLibrary so I am using that one to populate the data in the journalEntry view